src/main/cpp/src/vcf/vcf2binary.cc

```diff
@@ -156,8 +158,13 @@ void VCFReader::initialize(const char* filename,
 void VCFReader::add_reader() {
   assert(m_indexed_reader->nreaders == 0);      //no existing files are open
   assert(m_fptr == 0);  //normal file handle should be NULL
-  if (bcf_sr_add_reader(m_indexed_reader, m_name.c_str()) != 1)
-    throw VCF2BinaryException(std::string("Could not open file ")+m_name+" : " + bcf_sr_strerror(m_indexed_reader->errnum) + " (VCF/BCF files must be block compressed and indexed)");
+  errno = 0;
+  if (bcf_sr_add_reader(m_indexed_reader, m_name.c_str()) != 1) {
+    std::string errmsg =  (errno>0)?logger.format(" errno={1} ({2})", errno, std::strerror(errno)):"";
```

Comment:
Might be better here to either set errmsg to the errno information (if errno>0) or set it to "(VCF/BCF files must be block compressed and indexed)" and remove that string from line 166

Reply:
I did have that first. But, this issue only crops up when opening index files, so thought that some indication that the issue in the form of `VCF/BCF files must be...` is with index files may be OK. Let me know, I will make the change. Thanks.

Reply:
@mlathara, made the changes requested. Thanks.